contributing/release-schedule.md

```diff
@@ -18,6 +20,10 @@ The AMP runtime and extensions are provided through a variety of different _rele
 
 To determine if a PR has been included in any of the following release channels, look for the GitHub labels _PR Use: In Canary_, _PR Use: In Production_, or _PR Use: In LTS_ (see the section on [determining if your change is in a release](#Determining-if-your-change-is-in-a-release) for more details).
 
+[tip]
+For a quick visual overview of when a PR has reached a certain stage you can refer to the [interactive release calendar](https://amp.dev/documentation/guides-and-tutorials/contribute/vendor-contributions/release-schedule/).
```

Comment:
Would it make sense to rename the page from `release-schedule` to `interactive-release-calendar`?